src/Models/SavedToken.php

```diff
@@ -62,4 +70,21 @@ protected function parseAttributes($data)
         }
         return $data;
     }
+
+    /**
+     * Get the path to save the token to.
+     *
+     * @return string The file path for the token file.
+     * @throws \Terminus\Exceptions\TerminusException
+     */
+    protected function getPath()
+    {
+        $path = $this->getConfig()->get('tokens_dir');
+        $id = $this->id;
+        // Reality check to prevent stomping on the local filesystem if there is something wrong with the config.
+        if (!$this->id || !$path) {
+            throw new TerminusException('Could not save the machine token.');
```

Comment:
I'd like to be more explicit with the reason why the machine token cannot be created: "Could not save the machine token because it is malformed or the tokens directory does not exist"